lib/lib-storage/README.md

```diff
@@ -21,7 +21,7 @@ try {
     ], // optional tags
     queueSize: 4, // optional concurrency configuration
     partSize: 1024 * 1024 * 5, // optional size of each part, in bytes, at least 5MB
-    leavePartsOnError: false, // optional manually handle dropped parts
+    leavePartsOnError: true, // optional throw error on failed parts
```

Comment:
I think this flag name is confusing. I'm bad with naming but maybe change it to something like 
```suggestion
    throwOnPartsError: true, // optional throw error on failed parts
```

Reply:
I agree it's a bad name, but that would be an even more breaking change. maybe a todo for the future?

Reply:
Yes, you're right